### ISSUE TITLE
[Nerf] Vampiric Touch

### ISSUE THREAD

**Issue by Weqrex** [mangosR2/mangos#822]
Wowwiki says: [Vampiric Touch] 40.0% (per tick; totaling 200%) Patch 3.2.0
http://www.wowwiki.com/Spell_power_coefficient 

There are 6 photos, 3 with Shadowform (15% more of shadowdamage) and 3 without it. All these tests have been done without ‘’Darkness’’ (10% more shadow damage) and with VT rank 5. In the photos we take into account the red rectangles. 
If we have Shadowform, VT will be (850 + [SPELLPOWER x 2]) x 1.15
If we have not Shadowform, VT will be 850 + (SPELLPOWER x 2)
850 is the base damage of the VT rank5. And the 1.15 is the bonus of having Shadowform. 
1.  No shadowform, no spell power. 
   http://imageshack.us/photo/my-images/263/16998444.jpg/
   The damage is correct. 5 ticks, 170x5=850
2.  No spell power, and with shadowform.
   http://imageshack.us/photo/my-images/820/68872565.jpg/
   The damage is correct. 5 ticks, 195x5=975  850x1.15= 977 aproximately
3.  1010 spell power without shadowform.
   http://imageshack.us/photo/my-images/84/89592702.jpg/
   5 ticks, 473x5=2365. Drdamage says that every hit should do 574. So, 850 + [1010 x 2] = 2870 damage that it should do. 
4.  1010 spell power and shadowform.
   http://imageshack.us/photo/my-images/202/95371154.jpg/ 
   5 ticks, 543 x 5 = 2715 damage that it does
   [850 + (1010 x 2)] x 1.15 = 3300 damage that it should do. Drdamage says that too. 
   You can see that the criticals are nerf too. For instance, one of them did 1085 and it should do 1320. 
5.  3252 of spell power without shadowform.
   http://imageshack.us/photo/my-images/207/84915269.jpg/ 
   5 ticks, 1145x5=5725 damage that it does. But it should do: 850 + (3252x2) =7354
6.  3252 of spell power and shadowform.
   http://imageshack.us/photo/my-images/213/50485232.jpg/
   5 ticks of 1316, 1316 x 5 = 6580 damage that it does. 
   [850 + (3252 x 2)] x 1.15 = 8457.1 damage that it should do. 

To sum up, VT is doing less damage than it should do.

But if we check the DotBonus of the BD, we can see: 

entry   = 34914
direct_bonus =   0
dot_bonus =  0.4 
comments =  Priest - Vampiric Touch

So the problem is not there, but in fact VT is nerf.
